tests/pipeline/test_dividends.py

```diff
@@ -206,10 +206,11 @@ def setup(self, dates):
         amounts = [['NaN', 1, 15], ['NaN', 7, 13], ['NaN', 3, 1], ['NaN', 23]]
         cols = {
             PREVIOUS_ANNOUNCEMENT: self.get_sids_to_frames(
-                zip_with_dates, announcement_dates, date_intervals, dates
+                zip_with_dates, announcement_dates, date_intervals, dates,
+                'datetime64[ns]', 'NaN'
```

Comment:
Remind me: the dividend changes in here are because this PR is also the one that incorporates string support?